spacy/tests/test_cli_app.py

```diff
@@ -40,3 +42,52 @@ def test_benchmark_accuracy_alias():
     assert result_benchmark.stdout == result_evaluate.stdout.replace(
         "spacy evaluate", "spacy benchmark accuracy"
     )
+
+
+def test_debug_data_trainable_lemmatizer_cli():
+    nlp = English()
+    train_docs = [
+        Doc(nlp.vocab, words=["I", "like", "cats"], lemmas=["I", "like", "cat"]),
+        Doc(
+            nlp.vocab,
+            words=["Dogs", "are", "great", "too"],
+            lemmas=["dog", "be", "great", "too"],
+        ),
+    ]
+    dev_docs = [
+        Doc(nlp.vocab, words=["Cats", "are", "cute"], lemmas=["cat", "be", "cute"]),
+        Doc(nlp.vocab, words=["Pets", "are", "great"], lemmas=["pet", "be", "great"]),
+    ]
+    with make_tempdir() as d_in:
+        train_bin = DocBin(docs=train_docs)
+        train_bin.to_disk(d_in / "train.spacy")
+        dev_bin = DocBin(docs=dev_docs)
+        dev_bin.to_disk(d_in / "dev.spacy")
+        # `debug data` requires an input pipeline config
+        CliRunner().invoke(
+            app,
+            [
+                "init",
+                "config",
```

Comment:
I really like this test and the fact that it calls into `spacy init config` with `trainable_lemmatizer`, ensuring that workflow works well :-)